<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CZ4031 Database System Principles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxSchema">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1001</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Schema</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>981</width>
       <height>201</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="combo_databases">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Insert SQL Query</string>
    </property>
    <widget class="QTextEdit" name="in_query">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>470</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis Result(s)</string>
    </property>
    <widget class="QLabel" name="out_query">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>471</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
